### ISSUE TITLE
Tests fail on Windows 10

### ISSUE THREAD

**Issue by hankache** [salortiz/NativeLibs#14]
```
Welcome to Rakudo(tm) v2020.12-31-gf89863249.
Implementing the Raku(tm) programming language v6.d.
Built on MoarVM version 2020.12-16-g34f9d8c7a.
```
```
❯ zef install NativeLibs
===> Searching for: NativeLibs
===> Testing: NativeLibs:ver<0.0.8>:auth<github:salortiz>
[NativeLibs] No such method 'CALL-ME' for invocant of type
[NativeLibs] 'NativeCall::Types::Pointer'
[NativeLibs]   in block  at t/10-search.t line 37
[NativeLibs]   in block <unit> at t/10-search.t line 10
[NativeLibs] foo.c: In function 'main':
[NativeLibs] foo.c:4:21: warning: format '%lu' expects argument of type 'long unsigned int', but argument 2 has type 'long long unsigned int' [-Wformat=]
[NativeLibs]      printf("size: %lu", sizeof(int));
[NativeLibs]                    ~~^   ~~~~~~~~~~~
[NativeLibs]                    %I64u
[NativeLibs] foo.c:4:21: warning: format '%lu' expects argument of type 'long unsigned int', but argument 2 has type 'long long unsigned int' [-Wformat=]
[NativeLibs]      printf("size: %lu", sizeof(int));
[NativeLibs]                    ~~^   ~~~~~~~~~~~
[NativeLibs]                    %I64u
[NativeLibs] gcc: error: $@).a: No such file or directory
[NativeLibs] The spawned command 'gcc -shared -Wl,--out-implib,lib$(notdir $@).a  -O3 -DNDEBUG   -lshell32 -lws2_32 -lmswsock -lrpcrt4 -ladvapi32 -lpsapi -liphlpapi -luserenv -luser32 -o foo.dll foo.o' exited unsuccessfully (exit code: 1, signal: 0)
[NativeLibs]   in block <unit> at t/20-compile.t line 12
===> Testing [FAIL]: NativeLibs:ver<0.0.8>:auth<github:salortiz>
Aborting due to test failure: NativeLibs:ver<0.0.8>:auth<github:salortiz> (use --force-test to override)
```

**Comment by salortiz** [comment]
@hankache,
Can you please test with the latest commit?

**Comment by hankache** [comment]
@salortiz error is different, but it still fails
```
❯ zef install .
===> Testing: NativeLibs:ver<0.0.8>:auth<github:salortiz>
[NativeLibs] foo.c: In function 'main':
[NativeLibs] foo.c:4:21: warning: format '%lu' expects argument of type 'long unsigned int', but argument 2 has type 'long long unsigned int' [-Wformat=]
[NativeLibs]      printf("size: %lu", sizeof(int));
[NativeLibs]                    ~~^   ~~~~~~~~~~~
[NativeLibs]                    %I64u
[NativeLibs] foo.c:4:21: warning: format '%lu' expects argument of type 'long unsigned int', but argument 2 has type 'long long unsigned int' [-Wformat=]
[NativeLibs]      printf("size: %lu", sizeof(int));
[NativeLibs]                    ~~^   ~~~~~~~~~~~
[NativeLibs]                    %I64u
[NativeLibs] gcc: error: $@).a: No such file or directory
[NativeLibs] The spawned command 'gcc -shared -Wl,--out-implib,lib$(notdir $@).a  -O3 -DNDEBUG   -lshell32 -lws2_32 -lmswsock -lrpcrt4 -ladvapi32 -lpsapi -liphlpapi -luserenv -luser32 -o foo.dll foo.o' exited unsuccessfully (exit code: 1, signal: 0)
[NativeLibs]   in block <unit> at t/20-compile.t line 12
===> Testing [FAIL]: NativeLibs:ver<0.0.8>:auth<github:salortiz>
Aborting due to test failure: NativeLibs:ver<0.0.8>:auth<github:salortiz> (use --force-test to override)
```

**Comment by salortiz** [comment]
@hankache, I totally forgot the second part!
Can you give it another try?

**Comment by hankache** [comment]
@salortiz Great! there was one last failing test. I fixed it in #15 and increased the version number.
Thanks.
